modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -2949,109 +2949,425 @@ GAPI_FLUID_KERNEL(GFluidPhase, cv::gapi::core::GPhase, false)
     }
 };
 
-GAPI_FLUID_KERNEL(GFluidResize, cv::gapi::core::GResize, true)
-{
-    static const int Window = 1;
-    static const auto Kind = GFluidKernel::Kind::Resize;
-
-    constexpr static const int INTER_RESIZE_COEF_BITS = 11;
-    constexpr static const int INTER_RESIZE_COEF_SCALE = 1 << INTER_RESIZE_COEF_BITS;
-    constexpr static const short ONE = INTER_RESIZE_COEF_SCALE;
+CV_ALWAYS_INLINE void vertical_anyLPI(const uchar* src0, const uchar* src1,
+                                      uchar* tmp, const int inLength,
+                                      const short beta) {
+    constexpr int nlanes = static_cast<int>(v_uint8::nlanes);
 
-    struct ResizeUnit
-    {
-        short alpha0;
-        short alpha1;
-        int   s0;
-        int   s1;
-    };
+    const int half_nlanes = nlanes/2;
+    int w = 0;
+    for (;;) {
```

Comment:
Looks like `while(true)`.

Reply:
I would make it actually `while(flag_name)` cause some compilers/sanitizers might produce warnings here